src/libs/ReportUtils.js

```diff
@@ -985,6 +984,39 @@ function hasOutstandingIOU(report, currentUserLogin, iouReports) {
     return report.hasOutstandingIOU;
 }
 
+/**
+ * @param {Object} report
+ * @param {String} report.iouReportID
+ * @param {Object} iouReports
+ * @returns {null|Number}
+ */
+function getIOUTotal(report, iouReports) {
+    if (report.hasOutstandingIOU) {
+        const iouReport = (iouReports && iouReports[`${ONYXKEYS.COLLECTION.REPORT_IOUS}${report.iouReportID}`]) || null;
+        if (iouReport) {
+            return iouReport.total;
+        }
+    }
+    return null;
+}
+
+/**
+ * @param {Object} report
+ * @param {String} report.iouReportID
+ * @param {String} currentUserLogin
+ * @param {Object} iouReports
+ * @returns {null|Boolean}
+ */
+function isIOUOwnedByCurrentUser(report, currentUserLogin, iouReports) {
```

Comment:
NAB but we can lose the `iouReports &&` if we default to an object like this

```suggestion
function isIOUOwnedByCurrentUser(report, currentUserLogin, iouReports = {}) {
```